lumigator/frontend/src/components/experiments/LExperimentForm.vue

```diff
@@ -231,30 +231,50 @@ async function handleRunExperimentClicked() {
     system_prompt: experimentPrompt.value || defaultPrompt.value,
   }
 
-  const workflows = await experimentsService.createExperimentWithWorkflows(
+  const [experimentId, workflowResults] = await experimentsService.createExperimentWithWorkflows(
     experimentPayload,
     modelSelection.value.selectedModels,
   )
+  // Separate successful workflows and errors
+  const workflows = workflowResults.filter((r) => r.status === 'fulfilled').map((r) => r.value)
+  const failures = workflowResults
+    .filter((r) => r.status === 'rejected')
+    .map((r) => r.reason?.response?.data?.detail)
+
+  failures.forEach((msg: string) => {
+    toast.add({
+      severity: 'error',
+      summary: `Workflow failed to start`,
+      detail: `${msg}`,
+      messageicon: 'pi pi-exclamation-triangle',
+      group: 'br',
+      life: 6000,
+    } as ToastMessageOptions & { messageicon: string })
+  })
```

Comment:
do we want to create multiple toasts for each workflow, or just one with the failure messages combined via a `.join` or similar?

Reply:
I don't mind, but I'll leave that for later if possible as I wanted to make sure 'it works' in the UI after the frontend changes. I guess ideally you don't have loads of failing ones so maybe having one per error makes sense (1 error => 1 action to fix it). Which ever way it goes 👍🏼 🤷🏼 😄